freqtrade/strategy/interface.py

```diff
@@ -168,6 +168,10 @@ def start(self, *args, **kwargs):
                     raise OperationalException(
                         'freqAI is not enabled. '
                         'Please enable it in your config to use this strategy.')
+
+                def cleanup(self, *args, **kwargs):
```

Comment:
the name of this method should be aligned to the method you call in `ft_bot_cleanup()` - which is shutdown - not `cleanup()`.

Reply:
Good catch, originally called it cleanup but changed it to shutdown. Forgot about that part.